src/CoreWCF.Primitives/src/CoreWCF/IdentityModel/Policy/DefaultEvaluationContext.cs

```diff
@@ -1,8 +1,8 @@
 ﻿using CoreWCF.IdentityModel.Claims;
+using CoreWCF.Security;
```

Comment:
Why was CoreWCF.Security needed to be added when no other changes were made to this file?

Reply:
Merged 2 SecurityUtils as per previous comments to one file which stays inside CoreWCF.Security